.github/workflows/docker.yml

```diff
@@ -67,18 +69,36 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  aarch64-base:
+  aarch64-jupyter:
+    uses: ./.github/workflows/docker-build-test-upload.yml
+    with:
+      parentImage: ""
+      image: base-jupyter
+      platform: aarch64
+      runsOn: ARM64
+
+  x86_64-jupyter:
     uses: ./.github/workflows/docker-build-test-upload.yml
     with:
       parentImage: ""
+      image: base-jupyter
+      platform: x86_64
+      runsOn: ubuntu-latest
+
+  aarch64-base:
+    needs: [aarch64-jupyter]
+    uses: ./.github/workflows/docker-build-test-upload.yml
+    with:
+      parentImage: base-jupyter
       image: base-notebook
       platform: aarch64
       runsOn: ARM64
 
   x86_64-base:
+    needs: [x86_64-jupyter]
```

Comment:
I like the name suggested by @consideRatio.
Right now, places like this one make it difficult to understand what's going on.

Reply:
@kevin-bates if you don't mind, let's use `docker-stacks-foundation` instead of `jupyter-base`?